analyzer/src/funTest/assets/projects/synthetic/gradle-all-dependencies-expected-result-with-curations.yml

```diff
@@ -47,7 +47,7 @@ analyzer:
     variables: {}
     tool_versions: {}
   config:
-    allow_dynamic_versions: false
+    allow_dynamic_versions: true
```

Comment:
Any idea why this is `true` now? The default is `false`, and the only `OrtMainFunTest` which uses this file does not seem to modify the default...

Reply:
Maybe come local configuration of your is being taken into account, as the test runs the full `OrtMain`?

Reply:
Indeed, fixed. This should not be the case, I think I'll make a separate PR to decouple the test from the user configuration.